aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -22,21 +22,19 @@ class QConv2dInt8 final : public c10::OperatorKernel {
       int64_t groups,
       double output_scale,
       int64_t output_zero_point) {
-    TORCH_CHECK(
-        fbgemm::fbgemmSupportedCPU(), "Your CPU does not support FBGEMM.");
-    TORCH_CHECK(
-        act.ndimension() == 4,
-        "Activations are supposed to have 4 dimensions.");
-    TORCH_CHECK(stride.size() == 2, "2D convolution only");
-    TORCH_CHECK(padding.size() == 2, "2D convolution only");
-    TORCH_CHECK(dilation.size() == 2, "2D convolution only");
-    TORCH_CHECK(output_padding.size() == 2, "2D convolution only");
-    TORCH_CHECK(
-        (dilation[0] == 1 && dilation[1] == 1),
-        "Currently dilation should be 1");
-    TORCH_CHECK(
-        (output_padding[0] == 0 && output_padding[1] == 0),
-        "Currently output padding should be 0");
+    TORCH_CHECK(fbgemm::fbgemmSupportedCPU(),
+                "[QConv2dInt8] Your CPU does not support FBGEMM.");
```

Comment:
is this prefix for unittesting only? feels a bit sketchy to users. Can't we use stacktrace for testing?

Reply:
Yes, it is only for unittesting -- to make sure the errors are raised. The problem is that cpp raises `RuntimeError` no matter what error it is. I'm not sure how to make sure the correct error was raised except putting a flag in the TORCH_CHECK

Reply:
Removed it -- using a `np.assert_raise_regex`